code/nordvpn_indicator.py

```diff
@@ -93,17 +98,54 @@ def build_menu(self):
         """
         menu = gtk.Menu()
 
+        # Create a Connect submenu
+        menu_connect = gtk.Menu()
         item_connect = gtk.MenuItem('Connect')
-        item_connect.connect('activate', self.nordvpn.connect)
+        item_connect.set_submenu(menu_connect)
+
+        # First item is to connect automatically
+        item_connect_auto = gtk.MenuItem('Auto')
+        item_connect_auto.connect('activate', self.auto_connect_cb)
+        menu_connect.append(item_connect_auto)
+
         menu.append(item_connect)
 
+        countries = self.nordvpn.get_countries()
+        countries_menu = gtk.Menu()
+        item_connect_country = gtk.MenuItem('Countries')
+        item_connect_country.set_submenu(countries_menu)
+        for c in countries:
+            item = gtk.MenuItem(c)
+            item.connect('activate', self.country_connect_cb)
+            countries_menu.append(item)
+        menu_connect.append(item_connect_country)
+
         item_disconnect = gtk.MenuItem('Disconnect')
         item_disconnect.connect('activate', self.nordvpn.disconnect)
         menu.append(item_disconnect)
 
-        item_status_check = gtk.MenuItem('Status check')
-        item_status_check.connect('activate', self.nordvpn.status_check)
-        menu.append(item_status_check)
+        # Create a submenu for the connection status
+        menu_status = gtk.Menu()
+        item_status = gtk.MenuItem('Status')
+        item_status.set_submenu(menu_status)
+
+        # First item is to refresh the status
+        item_refresh = gtk.MenuItem('Refresh')
+        item_refresh.connect('activate', self.nordvpn.status_check)
+        menu_status.append(item_refresh)
+
+        # Add a label to show the current status details
+        self.status_label = gtk.MenuItem('')
+        menu_status.append(self.status_label)
+
+        menu.append(item_status)
+
+        # Create a submenu for the settings
+        menu_settings = gtk.Menu()
+        item_settings = gtk.MenuItem('Settings')
+        item_settings.set_submenu(menu_settings)
+        # TODO add settings
+        menu.append(item_settings)
```

Comment:
Maybe it's better not to include this? As Settings does nothing yet

Reply:
Yeah sorry!